templates/workspaces/base/terraform/variables.tf

```diff
@@ -8,6 +8,11 @@ variable "tre_resource_id" {
   description = "Resource ID"
 }
 
+variable "shared_storage_quota" {
```

Comment:
can you add a description to this variable please (other variables have descriptions).
I would suggest...

```Quota (in GB) to set for the VM Shared Storage.```

Reply:
Changes implemented